syntax = "proto3";
package system_map;





/*
GetSystem请求
*/
message GetSystemRequest {
    /*
    应用系统ID
    */
    string systemId = 1 ;
}

/*
GetSystem返回
*/
message GetSystemResponse {
    /*
    应用系统 ID
    */
    string instanceId = 1 ;
    /*
    名称
    */
    string name = 2 ;
    /*
    链接地址
    */
    string url = 3 ;
    /*
    图标
    */
    string icon = 4 ;
    /*
    类型
    */
    string category = 5 ;
    /*
    描述
    */
    string memo = 6 ;
}

/*
GetSystemApi返回
*/
message GetSystemResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    system_map.GetSystemResponse data = 4 ;
}

